chain-impl-mockchain/src/stake/controlled.rs

```diff
@@ -2,13 +2,14 @@ use crate::{
     account::{self, Identifier},
     rewards::Ratio,
     stake::Stake,
-    utxo,
+    tokens::identifier::TokenIdentifier,
 };
-use chain_addr::{Address, Kind};
 use imhamt::Hamt;
 use std::{collections::hash_map::DefaultHasher, fmt, num::NonZeroU64};
 
 #[derive(Default, Clone, Eq, PartialEq)]
+// TODO: rename this to WeightControl? VotingPowerControl?  It's a form of Stake, technically, but
+// I'm not sure about using the same term for two different things.
 pub struct StakeControl {
     assigned: Stake,
     unassigned: Stake,
```

Comment:
do we still have the concept of unassigned stake with tokens?

Reply:
I don't think so, not sure if we could in the future.